plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQuerySplitManager.java

```diff
@@ -163,7 +161,9 @@ private List<BigQuerySplit> readFromBigQuery(
 
         log.debug("readFromBigQuery(tableId=%s, projectedColumns=%s, filter=[%s])", remoteTableId, projectedColumns, filter);
         List<BigQueryColumnHandle> columns = projectedColumns.get();
-        List<String> projectedColumnsNames = new ArrayList<>(getProjectedColumnNames(columns));
+        List<String> projectedColumnsNames = getProjectedColumnNames(columns);
+        ImmutableList.Builder<BigQueryColumnHandle> additionalDomainColumns = ImmutableList.builder();
```

Comment:
nit: Can we rename the variable as `projectedColumns`

Reply:
We already have such variable, maybe `projectedAndConstraintColumns` ?

Reply:
How about `projectedColumnHandles`